Modix.Bot/Modules/PingModule.cs

```diff
@@ -86,46 +106,67 @@ public async Task Ping()
             });
         }
 
-        private string FormatLatency(double latency)
+        private string FormatLatency(double latency, string icon = "📶")
         {
-            if (latency == 0)
+            if (latency == -1)
             {
-                return "📶 Error ⚠️";
+                return $"{icon} Error ⚠️";
             }
 
             var suffix = latency switch
             {
-                _ when latency > 250 => "💔",
+                _ when latency > 300 => "💔",
                 _ when latency > 100 => "💛", //Yellow heart - trust me
                 _ when latency < 100 => "💚", //Green heart - trust me again
                 _ => "❓"
```

Comment:
Will this just end up printing the ❓ icon if the latency is exactly 100ms?

Reply:
Oops, good call. Will fix.